.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode/
```

Comment:
Please remove these directories. I guess they are specific to your environment, but not everyone use the same.

Reply:
VSCode is commonly used, and these are pretty standard ignore directories, but ok, i remove them